ofl/yrsa/upstream.yaml

```diff
@@ -0,0 +1,16 @@
+branch: master
+files:
+  OFL.txt: OFL.txt
+  documentation/Yrsa-DESCRIPTION.en_us.html: DESCRIPTION.en_us.html
+  fonts/YrsaVariable/YrsaVF-Ups.ttf: Yrsa[wght].ttf
+  fonts/YrsaVariable/YrsaVF-Its.ttf: Yrsa-Italic[wght].ttf
+  fonts/Yrsa/Yrsa-Light.ttf: static/Yrsa-Light.ttf
```

Comment:
Was the family hand hinted?

Reply:
The current sources are not hand hinted

Reply:
Then no static fonts are needed

Reply:
> Then no static fonts are needed

Considering #2616, are you not putting the cart before the horse? As long as the API is not able to produce usable static fonts, I find that on the contrary having them here is essential.